spec/models/shop_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+describe 'Shop model' do
+  let(:shop) { FactoryBot.create(:shop) }
```

Comment:
use factory for this
you need to test the relations also
structure should be
describe relations ....
  describe 
  describe
describe validation ...
  describe
  describe